mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -27,14 +27,16 @@
 
 	<div class="welcome-message">
 		<liferay-ui:message key="welcome" />
+
 		<%= user.getFirstName() %>
 
 		<img alt="<%= user.getFullName() %>" src="<%=user.getPortraitURL(themeDisplay) %>">
 	</div>
 
-	<liferay-ui:tabs names="task-list-tab,task-history-tab,finished-tasks-tab" refresh="false">
+	<liferay-ui:tabs names="task-list-tab,task-history-tab,profile-tab,contact-tab" refresh="false">
 		<liferay-ui:section>
 			<div class="task-container">
+
```

Comment:
Repeat of issue from https://github.com/markocikos/liferay-internship/pull/152#discussion_r125622095

Two base rules of formatting HTML, as mentioned before:
1. Child node should indented by tab, no blank row.
1. Sibling nodes should be separated by a blank row.

It was fine as it was.